lib/aff4_objects/security.py

```diff
@@ -151,10 +151,11 @@ class ApprovalWithApproversAndReason(Approval):
 
   class SchemaCls(Approval.SchemaCls):
     """The Schema for the ClientAccessApproval class."""
+
     LIFETIME = aff4.Attribute(
         "aff4:approval/lifetime", rdfvalue.RDFInteger,
-        "The number of microseconds an approval is valid for.",
-        default=4 * 7 * 24 * 60 * 60 * 1000000)  # 4 weeks
+        "The number of seconds an approval is valid for.",
```

Comment:
I think this should probably be an rdfvalue.Duration so you can specify "1w" and it will handle seconds/microseconds properly in comparison and arithmetic operations.  You'll have to test what happens to old approvals that are stored with the old format though.